pennylane/devices/tests/test_properties.py

```diff
@@ -177,19 +174,6 @@ def test_passthru_interface_is_correct(self, device_kwargs):
             else:
                 pytest.skip("Cannot import torch")
 
-    def test_provides_jacobian(self, device_kwargs):
```

Comment:
Pretty sure no device actually provides the jacobian, and I can't find anything about a `jacobian` attribute for the qnode.